e2e/transaction_test.go

```diff
@@ -124,7 +125,7 @@ func TestEthTransfer(t *testing.T) {
 		config.SetConsensus(framework.ConsensusDev)
 		for _, acc := range validAccounts {
 			config.Premine(acc.address, acc.balance)
-			config.SetBaseFeeConfig("")
+			config.SetBurnContract(types.StringToAddress("0xBurnContract"))
```

Comment:
Would be nicer if we would go with an address in proper format (e.g. `types.ZeroAddress`), but ok, your call.